iac/tf-multienv-cicd-anthos-autopilot/env-staging.tf

```diff
@@ -44,7 +44,6 @@ module "gke_staging" {
   horizontal_pod_autoscaling      = true
   create_service_account          = false # currently not supported by terraform for autopilot clusters
   cluster_resource_labels         = { "mesh_id" : "proj-${data.google_project.project.number}" }
-  datapath_provider               = "ADVANCED_DATAPATH"
```

Comment:
Thanks for removing the use of `datapath_provider`.
Looks like all checks are passing. 👍 
Google Cloud documentation isn't _super_ clear about this `datapath_provider` field, but I'm guessing it was a no-op for us because we use autopilot, and autopilot comes with dataplane v2.

[Related documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/dataplane-v2#create-cluster):
> Warning: GKE Dataplane V2 comes with [Kubernetes network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/) enforcement built-in. This means that you don't need to [enable network policy](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy#enabling_network_policy_enforcement) in clusters that use GKE Dataplane V2. If you try to explicitly enable or disable network policy enforcement in a cluster that uses GKE Dataplane V2, the request will fail with the error message Enabling NetworkPolicy for clusters with DatapathProvider=ADVANCED_DATAPATH is not allowed..

Reply:
That was my take on it too!